notifications/reactive/src/main/java/org/trellisldp/reactive/ReactiveEventService.java

```diff
@@ -0,0 +1,60 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.trellisldp.reactive;
+
+import io.reactivex.BackpressureStrategy;
+import io.reactivex.Flowable;
+import io.reactivex.subjects.PublishSubject;
+
+import javax.enterprise.context.ApplicationScoped;
+import javax.inject.Inject;
+
+import org.eclipse.microprofile.reactive.messaging.Message;
+import org.eclipse.microprofile.reactive.messaging.Outgoing;
+import org.trellisldp.api.*;
+
+/**
+ * A microprofile-based event service.
+ */
+@ApplicationScoped
+public class ReactiveEventService implements EventService {
+
+    public static final String REACTIVE_DESTINATION = "trellis";
+
+    private final ActivityStreamService serializer;
+    private final PublishSubject<Message<String>> subject = PublishSubject.create();
+
+    /**
+     * Create a new Kafka Event Service.
```

Comment:
Is this "Kafka" left over from some cut-and-paste?

Reply:
that's now fixed.